internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1098,7 +1098,10 @@ func selectClusterNetworkType(params *models.V2ClusterUpdateParams, cluster *com
 
 func (r *ClusterDeploymentsReconciler) updateInstallConfigOverrides(ctx context.Context, log logrus.FieldLogger, clusterInstall *hiveext.AgentClusterInstall,
 	cluster *common.Cluster) error {
-	// handle InstallConfigOverrides
+	// not relevant for day2 cluster - install config is not used, pulled from installed cluster
```

Comment:
```suggestion
	// not relevant for day2 cluster - install config is not used
```

day-2 installation only really requires a single ignition file from MCS, it doesn't invoke the openshift-installer at any point so there's no need for an install-config

Reply:
thanks for the clarification